publish.sh

```diff
@@ -1,4 +1,105 @@
+git push origin main $VERSION
+
+echo "Pushed \"main\" and \"$VERSION\" to repo."
+
+curl \
+  -u bitAnalyticsCircleCi:$ANALYTICS_GH_TOKEN \
+  -X POST \
+  -H "Accept: application/vnd.github.v3+json" \
+  https://api.github.com/repos/bitmovin/bitmovin-analytics-collector-android/releases \
+  -d "{\"tag_name\":\"v$VERSION\", \"name\": \"v$VERSION\", \"draft\": false}"
+
+echo "Created release in public repo."
+
+echo "Creating and publishing :collector project..."
+./gradlew -PdevelopLocal=false :collector:clean
+./gradlew -PdevelopLocal=false :collector:build
+./gradlew -PdevelopLocal=false :collector:assembleRelease
+./gradlew -PdevelopLocal=false :collector:artifactoryPublish
+echo "Created and published :collector project."
+
+echo "Creating and publishing :collector-bitmovin-player project..."
+./gradlew -PdevelopLocal=false :collector-bitmovin-player:clean
+./gradlew -PdevelopLocal=false :collector-bitmovin-player:build
+./gradlew -PdevelopLocal=false :collector-bitmovin-player:assembleRelease
+./gradlew -PdevelopLocal=false :collector-bitmovin-player:artifactoryPublish
+echo "Created and published :collector-bitmovin-player project."
+
+echo "Creating and publishing :collector-exoplayer project..."
+./gradlew -PdevelopLocal=false :collector-exoplayer:clean
+./gradlew -PdevelopLocal=false :collector-exoplayer:build
+./gradlew -PdevelopLocal=false :collector-exoplayer:assembleRelease
+./gradlew -PdevelopLocal=false :collector-exoplayer:artifactoryPublish
+echo "Created and published :collector-exoplayer project."
+
+file="./bitmovin.properties"
+
+if [ -f "$file" ]
+then
+  echo "$file found."
+
+  while IFS='=' read -r key value
+  do
+    key=$(echo $key | tr '.' '_')
+    eval ${key}=\${value}
+  done < "$file"
+else
+  echo "$file not found."
+fi
+
+echo "Distributing the artifacts to bintray..."
+
+curl \
+  -u ${artifactoryUser}:${artifactoryPassword} \
+  -X POST \
+  https://bitmovin.jfrog.io/artifactory/api/distribute \
+  -H "Content-Type: application/json" \
+  -d "{
+    \"targetRepo\": \"releases\",
+    \"overrideExistingFiles\": true,
+    \"packagesRepoPaths\": [
+      \"libs-release-local/com/bitmovin/analytics/collector/$VERSION/collector-$VERSION.pom\",
+      \"libs-release-local/com/bitmovin/analytics/collector/$VERSION/collector-$VERSION.aar\",
+      \"libs-release-local/com/bitmovin/analytics/collector-bitmovin-player/$VERSION/collector-bitmovin-player-$VERSION.pom\",
+      \"libs-release-local/com/bitmovin/analytics/collector-bitmovin-player/$VERSION/collector-bitmovin-player-$VERSION.aar\",
+      \"libs-release-local/com/bitmovin/analytics/collector-exoplayer/$VERSION/collector-exoplayer-$VERSION.pom\",
+      \"libs-release-local/com/bitmovin/analytics/collector-exoplayer/$VERSION/collector-exoplayer-$VERSION.aar\"
+    ]}"
+
+echo "\nDistributed the artifacts to bintray."
+
+echo "Don't forget to update the changelog in Contentful."
+echo "https://app.contentful.com/spaces/blfijbdi3ei3/entries"
```

Comment:
Can't we just open this directly?

`open "https://app.contentful.com/spaces/blfijbdi3ei3/entries"`